pkg/server/server.go

```diff
@@ -14,9 +14,9 @@ import (
 	"github.com/coreos/pkg/capnslog"
 	"github.com/coreos/pkg/health"
 
-	"github.com/openshift/console/auth"
+	"github.com/openshift/console/pkg/auth"
 	"github.com/openshift/console/pkg/proxy"
-	"github.com/openshift/console/version"
+	"github.com/openshift/console/pkg/version"
```

Comment:
We need to update at least the `build-backend.sh` script if we move this.

Reply:
Also `test-backend.sh`. We should sweep the scripts to see if there are others this would cause problems for.

Reply:
Updating:

```bash
LD_FLAGS="-w -X github.com/openshift/console/version.Version=${GIT_TAG}"
LD_FLAGS="-w -X github.com/openshift/console/pkg/version.Version=${GIT_TAG}"
```

Reply:
Updated.
That file is all of `var Version string`, heh.